### PR TITLE
add h1 element

### PR DESCRIPTION
**Pull request by necr0coder**
add h1 element

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>test</title>
   </head>
   <body>
-    
+    <h1>hello world!</h1>
```
Comment (jrhurleycode) [inline index.html]: i don't like this

**Review by jrhurleycode — CHANGES_REQUESTED** [review]
yeah this looks good

File: index.html (hunk 1 above)
Comment (necr0coder) [reply index.html]: ok cool

**Review by jrhurleycode — COMMENTED** [review]
yeah looks good

**Comment by jrhurleycode** [comment]
asdf

**Review by jrhurleycode — APPROVED** [review]

**Pull request closed without merging**